Transfer/Sources/CreateTransferPresenter.swift

```diff
@@ -143,8 +143,6 @@ final class CreateTransferPresenter {
     }
 
     private func resetValues() {
-        amount = "0"
-        notes = nil
         didTapTransferAllFunds = false
         availableBalance = nil
```

Comment:
didTapTransferAllFunds and availableBalance are also part of this ticket. So I would suggest simply delete resetValues(). Also update the logic to set amount = availableBalance if didTapTransferAllFunds is true once availableBalance is fetched.

Reply:
Will check it